test/validators/custom-styles-validator.test.ts

```diff
@@ -0,0 +1,83 @@
+import * as path from 'path';
+import { CustomStylesValidator } from '../../lib/validators/custom-styles-validator';
+
+describe('CustomStylesValidator', () => {
+    let definition: any;
+    let error: jasmine.Spy;
+    let validator: CustomStylesValidator;
+
+    beforeEach(() => {
+        // valid definition
+        definition = {
+            customStyles: [],
+        };
+        error = jasmine.createSpy('error');
+    });
+
+    describe('validate', () => {
+        it('should do nothing when no custom styles are given', async () => {
+            delete definition.customStyles;
+            validator = new CustomStylesValidator(error, definition, new Set());
+            await validator.validate();
+            expect(error).not.toHaveBeenCalled();
+        });
+
+        it('should do nothing when no custom styles are given', async () => {
```

Comment:
Same test description as previous one, but this one is for an empty array.

Reply:
Ok, only the empty array one should be tested. The `parseDefinition` makes sure it is always an array. So this should be fine.